src/Yesod/Auth/OAuth2/ORCID.hs

```diff
@@ -0,0 +1,50 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Yesod.Auth.OAuth2.ORCID
+  ( oauth2Orcid
```

Comment:
Thanks for correcting the module name. I think I'd like the capitalization to be consistent everywhere.

```suggestion
  ( oauth2ORCID
```

Reply:
Done